Source/WebGPU/WGSL/ConstantFunctions.h

```diff
@@ -822,9 +822,9 @@ CONSTANT_FUNCTION(Cross)
         auto v2 = std::get<T>(rhs.elements[2]);
 
         ConstantVector result(3);
```

Comment:
I think that would be a problem since a ConstantValue is a variant which must include ConstantVector. I think it needs to be a runtime check, but I'm happy to consider a different approach